sdk/storage/azure-storage-common/test/crypt_functions_test.cpp

```diff
@@ -15,16 +15,6 @@ namespace Azure { namespace Storage { namespace Test {
     return std::vector<uint8_t>(start, start + strlen(text));
   }
 
-  TEST(CryptFunctionsTest, Sha256)
-  {
-    EXPECT_EQ(
-        Azure::Core::Convert::Base64Encode(_internal::Sha256(ToBinaryVector(""))),
-        "47DEQpj8HBSa+/TImW+5JCeuQeRkm5NMpJWZG3hSuFU=");
-    EXPECT_EQ(
-        Azure::Core::Convert::Base64Encode(_internal::Sha256(ToBinaryVector("Hello Azure!"))),
-        "Mjzwx2mqGHb9FSgjm33ShNmXYndkgvwA6tQmEiskOHg=");
-  }
```

Comment:
Do we have tests and are we confident that the `_internal::Sha256` that is being removed and the new `Azure::Core::Cryptography::_internal::Sha256Hash` being used instead, have the same semantics and behavior?

I recommend keeping this test case, and just porting it to use the new one.

Reply:
I don't think that'd be necessary. SHA-256 in very standardized. It'd be horrible if we compute hash with two implementations and get different results.